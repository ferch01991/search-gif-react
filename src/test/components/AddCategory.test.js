const { shallow } = require("enzyme")
const { AddCategory } = require("../../AddCategory")

describe('Prueba en AddCategory', () => {
    
    const setCategory = () => {};
    const wrapper = shallow(<AddCategory setCategory={setCategory} />);

    test('should show correctily', () => {

        expect( wrapper ).toMatchSnapshot();
    })

    test('should change the text box', () => {
        const input = wrapper.find('input');
        const value = 'Hola mundo'

        input.simulate('change', { target: {value}})

        expect( wrapper.find('p').text().trim() ).toBe(value)
    })
    
    


})
