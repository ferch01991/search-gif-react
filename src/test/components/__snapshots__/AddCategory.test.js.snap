// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en AddCategory should show correctily 1`] = `
<form
  onSubmit={[Function]}
>
  <p>
     
     
  </p>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
